### PR TITLE
Setup Linters Checkers 

### PR DESCRIPTION
**Pull request by unorjikingsley**
## I implemented the following in this PR: 

  > - Added Linters checkers to my code 
  > - Added a descriptive README file
  > - Created a React application using [Create React App](https://create-react-app.dev/)

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
## Hi @unorjikingsley, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: 

> **You have submitted a project, that contains the following issues:**
  > - PR does not have a summary
  > - README is not descriptive. You didn't use the correct template
  > - PR title is not descriptive
  > - The project branch name is not descriptive
  > - Kindly check how to write good commit messages in the general requirements of each project

Please kindly take your time, and document your work in a professional manner before submitting it for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by wayungi — COMMENTED** [review]
Hi @unorjikingsley , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [x] You have submitted a project with a readme having a live demo section without the link. Please delete the live demo section if the project is not deployed. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ibizugbe — COMMENTED** [review]
Hi @unorjikingsley 👋👋👋, 

INVALID


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the readme features of the [microverse readme template](https://github.com/microverseinc/readme-template). Kindly make use of this [template ](https://github.com/microverseinc/readme-template)to document your readme and ensure the instructions for using the [template](https://github.com/microverseinc/readme-template) is obeyed😉. Kindly ensure every section in the template is filled except the FAQ section which is optional😉 and ensure the table of content is also intact👍.. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,70 +1,196 @@
...
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# 📗 Table of Contents
 
-## Available Scripts
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
```
Comment (Olamarx) [inline README.md:8]: - [ ] I will implore you to add the Key Features as it is a requirement.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,196 @@
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ -->
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Olamarx) [inline README.md]: - [ ] Kindly fix the link, it goes to 404 page.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @unorjikingsley, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the Key features, kindly add it for your readme to be professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mikethreels — APPROVED** [review]
Hi Kingsley, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job setting up the project.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**